### DISCUSSION TITLE
How to store geolocation types please

### DISCUSSION THREAD

**Discussion by hariprasadms** [AntlerVC/firetable#479] [Support / Q&A] [answered]
Hi, I am looking to store geolocation details. Looks like there is no data type yet available in firetable as of now. Please suggest how to do this.

**Comment by bofeiw** [comment]
I'd recommend to try to use JSON as a temporary alternative solution.

**Comment by harinij** [comment, marked as answer]
We have introduced a new field now for Geolocation in the recent release. Check that out.
